src/xo-app/new-vm/index.js

```diff
@@ -83,6 +89,12 @@ export default class NewVm extends BaseComponent {
     this.state = { state: {} }
   }
 
+  getPoolNetworks = createSelector(
+    () => this.props.networks,
+    () => this.state.state.pool && this.state.state.pool.id,
```

Comment:
You could use a intermediary variable, I feel I'm stuttering when I'm reading this line.